endpoints/openrtb2/auction.go

```diff
@@ -475,13 +505,38 @@ func validateNativeAsset(asset nativeRequests.Asset, impIndex int, assetIndex in
 	return nil
 }
 
+func validateNativeEventTrackers(trackers []nativeRequests.EventTracker, impIndex int) error {
+	for i := 0; i < len(trackers); i++ {
+		if err := validateNativeEventTracker(trackers[i], impIndex, i); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
 func validateNativeAssetTitle(title *nativeRequests.Title, impIndex int, assetIndex int) error {
 	if title.Len < 1 {
 		return fmt.Errorf("request.imp[%d].native.request.assets[%d].title.len must be a positive integer", impIndex, assetIndex)
 	}
 	return nil
 }
 
+func validateNativeEventTracker(tracker nativeRequests.EventTracker, impIndex int, eventIndex int) error {
+	if tracker.Event < 1 || tracker.Event > 4 {
```

Comment:
I know these are defined in the spec but maybe 
```go
if tracker.Event < native.EventTypeImpression || tracker.Event > native.EventTypeViewableVideo50 {
```
and similarly in other places using the named constants.

Reply:
akh... good call. Will fix